discord/client.py

```diff
@@ -3039,3 +3045,100 @@ def persistent_views(self) -> Sequence[View]:
         .. versionadded:: 2.0
         """
         return self._connection.persistent_views
+
+    async def create_emoji(
+        self,
+        *,
+        name: str,
+        image: bytes,
+    ) -> Emoji:
+        """|coro|
+
+        Create an emoji for the current application.
+
+        .. versionadded:: 2.5
+
+        Parameters
+        ----------
+        name: :class:`str`
+            The emoji name. Must be at least 2 characters.
+        image: :class:`bytes`
+            The :term:`py:bytes-like object` representing the image data to use.
+            Only JPG, PNG and GIF images are supported.
+
+        Raises
+        ------
+        MissingApplicationID
+            The application ID could not be found.
+        HTTPException
+            Creating the emoji failed.
+
+        Returns
+        -------
+        :class:`.Emoji`
+            The emoji that was created.
+        """
+        if self.application_id is None:
+            raise MissingApplicationID
+
+        img = utils._bytes_to_base64_data(image)
+        data = await self.http.create_application_emoji(self.application_id, name, img)
+        guild = self._connection._get_or_create_unavailable_guild(0)
+        return Emoji(guild=guild, state=self._connection, data=data)
+
+    async def fetch_emoji(self, emoji_id: int) -> Emoji:
```

Comment:
I guess ``emoji_id`` should be positional only as the other ``fetch_``/``get_`` methods.

```suggestion
    async def fetch_emoji(self, emoji_id: int, /) -> Emoji:
```

Reply:
Good catch, ty